conventions/documentation/create.md

```diff
@@ -6,53 +6,99 @@ parent: Conventions
+- Name of resource is included on button.
+- Name of resource is not included in methods to create it.
+- Methods are recommended to start with “With” or “From”.
+- “From YAML” should appear last, when applicable.
+
+![Dropdown create button multimethod 2](../images/create-button-dropdown-multiMethod2.png)
+
+![Dropdown create button multimethod 1](../images/create-button-dropdown-multiMethod1.png)
+
+**Multiple objects created from one button**
+- Button just reads “Create”.
+- Names of resources are listed as options.
+
+![Dropdown create button multiresource](../images/create-button-dropdown-multiResource.png)
+
+**Multiple methods to create multiple objects**
+- Button just reads “Create”.
+- Groups are used to separate resource types, with the methods in each group.
+- A Separator component can appear between resource types.
+
+![Dropdown create button multiresourcemethod](../images/create-button-dropdown-multiResourceMethod.png)
+
+---
+
+### Create buttons in selection fields
+
+*Fields that allow the user to optionally create a resource where they would otherwise be selecting one have a create action as the first option in the dropdown, seperated by a divider.*
+![Create from selection](../images/create-button-from-selection.png)
+
+---
+
+## Creation with form/YAML
+
+### Creation via YAML only:
 
 *Some resource types can only be created with YAML. The first time viewing the YAML creation, the samples panel should be open by default.*
 ![import YAML](../images/create-yaml-only1.png)
 ![import YAML](../images/create-yaml-only2.png)
 
 ---
 
-**Create from button (Form only):**
+### Creation via form only:
 
-*Some resource types can only be created with a form. After the resource is created, the user is automatically navigated to the created resource.*
+*Some resource types can only be created with a form.*
 ![import YAML](../images/create-form-only1.png)
 ![import YAML](../images/create-form-only2.png)
 ![import YAML](../images/create-form-only3.png)
 
 ---
 
-**Create from button (Form with option for YAML):**
+### Creation via form with option for YAML:
 
-*Some resource types can be created with a form (as the default) with an option to switch to create with YAML. Once switched to the YAML editor changes from the form editor are lost and it is not possible to switch back to the form view. Ideally this switch is confirmed with the user. After the resource is created, the user is automatically navigated to the created resource.*
+*Some resource types can be created with a form (as the default) with an option to switch to create with YAML. Once switched to the YAML editor changes from the form editor are lost and it is not possible to switch back to the form view. Ideally this switch is confirmed with the user.*
 ![import YAML](../images/create-form-then-yaml1.png)
 ![import YAML](../images/create-form-then-yaml2.png)
 ![import YAML](../images/create-form-then-yaml3.png)
 ![import YAML](../images/create-form-then-yaml4.png)
 
```

Comment:
This is tied to my comment above about the first section, but I think it becomes harder to follow along once we get into these images without text in-between because I don't understand the context. 

Also, this is more of a FYI but we decided on the dev side to try to start consistently using the pink pointer when appropriate. The when appropriate part is totally up to you haha but I find it most useful when I'm referencing an area of the ui that requires a selection. I think the full-screen images could be a good candidate.